cpp/src/grammar.cpp

```diff
@@ -127,15 +120,16 @@ Grammar::Grammar() : emerald_parser(syntax) {
       std::vector<std::string> class_names = repeated<std::string>(sv, 2);
       NodePtr body = present(sv, 3) ? sv[3].get<NodePtr>() : NodePtr();
       NodePtr attributes = present(sv, 4) ? sv[4].get<NodePtr>() : NodePtr();
+      NodePtr nested =
+        present(sv, 5) ? sv[5].get<peg::SemanticValues>()[0].get<NodePtr>() : NodePtr();
 
-      return NodePtr(new TagStatement(tag_name, id_name, class_names, body, attributes));
+      return NodePtr(
+        new TagStatement(tag_name, id_name, class_names, body, attributes, nested));
```

Comment:
I also added another parameter to the list, btw, which is what made me think to bump it down a line